recipes/r-bart/meta.yaml

```
@@ -0,0 +1,92 @@
+{% set version = '2.9.6' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-bart
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/BART_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/BART/BART_{{ version }}.tar.gz
+  sha256: 6e23eac476e894c0e6540eaa80e8436ea04261ce6aaac8d4f6f8f7ceeb082e0f
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}sed               # [win]
+    - {{ posix }}grep              # [win]
+    - {{ posix }}autoconf
+    - {{ posix }}automake          # [not win]
+    - {{ posix }}automake-wrapper  # [win]
+    - {{ posix }}pkg-config
+    - {{ posix }}make
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
```

OpenMP declarations go in `build` (they have *strong_exports*):

```suggestion
    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
    - libgomp                      # [linux]
    - llvm-openmp                  # [osx]
```